_includes/component/social-links.html

```diff
@@ -0,0 +1,11 @@
+<ul class="list-unstyled mb-0">
+  {% for item in site.data.social %}
+  {% if item.visible == true %}
+  <li>
+    <a class="text-light" href="{{ item.link }}" title="{{ item.username }} on {{ item.provider.displayName }}">
```

Comment:
`target="_blank"` is missing from the attribute list

Reply:
This was intentional, but I forgot to call it out and add a justification.

1. It's [not a friendly practice](https://css-tricks.com/use-target_blank/) to override default browser behavior this way for links, especially on a marketing website or blog (in an app opening an external link, maybe). It's like holding the user hostage on our blog.

2. Our own marketing website (www.godaddy.com) leaves the default behavior in place for the footer social links.

3. Finally, considering that our target visitors are developers/designers/sysadmins, they likely won't won't appreciate overriding expected browser behaviors.

If you are firmly against the change I'll yank it, but it may be nice to get consensus first because I feel the majority of people would be bothered by this behavior.